_transforms.py

```diff
@@ -27,3 +27,19 @@ def parse_attachment_url(obj):
 
     # format as markdown
     return f"[Attachment]({url}) ({size})"
+
+
+def parse_email(email_addr):
+    """
+    Extract `<First name> <Last intial>.` from email address
+    """
+    try:
+        first, last = email_addr.split("@")[0].split(".")
```

Comment:
I'll have to remember to unpack variables like this in the future. 😎

Reply:
thanks @mddilley !